tests/parser/features/test_assert_unreachable.py

```diff
@@ -32,13 +28,9 @@ def foo(val: int128) -> bool:
 
     assert c.foo(2) is True
 
-    assert_tx_failed(lambda: c.foo(1))
-    assert_tx_failed(lambda: c.foo(-1))
-
-    with pytest.raises(TransactionFailed) as e_info:
-        c.foo(-2)
-
-    assert "Invalid opcode 0xfe" in e_info.value.args[0]
```

Comment:
Side note: hitting the "invalid opcode" causes the transaction to revert, and use all possible gas. Which is probably why the transaction wouldn't estimate a gas cost for itself (it was never successfully able to avoid using all the gas).

This is a fine solution to that problem (setting gas to 50k)

Reply:
Update: With gas estimates now an optional flag, I have reverted the gas parameter to be calculated based on the original logic of the gas estimate + 50000 gas. It is therefore now unnecessary to manually supply the `gas` argument as a kwarg.